allure-generator/src/main/javascript/helpers/timeago.js

```diff
@@ -0,0 +1,8 @@
+import moment from 'moment';
+import runtime from 'injectify/runtime';
```

Comment:
Replace this with `handlebars/runtime`.

Injectify is a kind of semi-internal Yandex thing, we doin't need this